designs/2024-repo-ecosystem-plugin-tests/README.md

```diff
@@ -103,30 +103,33 @@ Otherwise the ESLint repository will assume only supporting up to the currently
 
 The plugins that will be included to start will be:
 
+- All `@eslint/*` plugins, including [`@eslint/css`](https://www.npmjs.com/package/@eslint/css), [`@eslint/json`](https://www.npmjs.com/package/@eslint/json), and [`@eslint/markdown`](https://www.npmjs.com/package/@eslint/markdown)
 - [`eslint-plugin-eslint-comments`](https://github.com/eslint-community/eslint-plugin-eslint-comments): to capture an `eslint-community` project and AST edge cases around comments
 - [`eslint-plugin-unicorn`](https://github.com/sindresorhus/eslint-plugin-unicorn): to capture a large selection of miscellaneous rules
 - [`eslint-plugin-vue`](https://github.com/vuejs/eslint-plugin-vue): to capture support for a framework with nested parsing of a non-JavaScript/TypeScript-standard syntax
 - [`typescript-eslint`](https://github.com/typescript-eslint/typescript-eslint): to capture testing TypeScript APIs and intricate uses of parsing in general
 
-Plugins will be selectively added if they meet all of the following criteria:
+Third-party plugins will be selectively added if they meet all of the following criteria:
 
 - &gt;1 million npm downloads a week: arbitrary large size threshold to avoid small packages
 - Adding a notable new API usage not yet covered: to avoid duplicate equivalent plugins
 - Has had a breakage reported on ESLint: to be cautious in adding to the list
 - Is under active maintenance and has taken a week or less to fix any ESLint breakages within the last year: to avoid packages that won't be updated quickly on failures
 
-The number of plugins should remain small.
-Each added plugin brings adds the risk of third-party breakage, so plugins will only be added after filing a new issue and gaining team consensus.
+The number of third-party plugins should remain small.
+Each added plugin brings adds a risk of breakage, so plugins will only be added after filing a new issue and gaining team consensus.
 
 ### Rollout
 
 This RFC expects the added ecosystem CI job to _likely_ consistently pass.
-However, to be safe, this RFC proposes adding a CI job in three steps:
+A CI job will be added to the `eslint/eslint` issue, but will not immediately be a part of `main` branch or PR branch builds.
+To be safe, this RFC proposes rolling out CI job in three steps:
 
-1. On a branch that is manually updated from `main` several times a week
+1. On a CI cron job once a day, targeting the `main` branch but not blocking its builds
 2. On the `main` branch only
 3. On all PRs targeting the `main` branch, alongside existing CI jobs
 
+Starting with a job separately from `main` ensures that unexpectedly high frequencies of breakages are caught early, without blocking `main` branch builds.
```

Comment:
I'm a bit confused by this process. Is the intent that we end up with all three at the end? Or just number 3?

And I'm unclear on the value of number 2. Presumably, this is meant to run only after a PR is merged, but how will we be notified if the job fails?